packages/geo/src/lib/datasource/shared/datasources/wfs-datasource.ts

```diff
@@ -2,139 +2,97 @@ import olSourceVector from 'ol/source/Vector';
 import * as OlLoadingStrategy from 'ol/loadingstrategy';
 import * as OlFormat from 'ol/format';
 
-import { uuid } from '@igo2/utils';
-
 import { DataSource } from './datasource';
 import { WFSDataSourceOptions } from './wfs-datasource.interface';
 import { WFSService } from './wfs.service';
 
 import { OgcFilterWriter } from '../../../filter/shared/ogc-filter';
 import { OgcFilterableDataSourceOptions } from '../../../filter/shared/ogc-filter.interface';
+import { formatWFSQueryString, defaultWfsVersion, defaultFieldNameGeometry, defaultMaxFeatures, gmlRegex, jsonRegex, checkWfsParams} from './wms-wfs.utils';
 
 export class WFSDataSource extends DataSource {
   public ol: olSourceVector;
-  public ogcFilterWriter: OgcFilterWriter;
 
   constructor(
     public options: WFSDataSourceOptions,
     protected wfsService: WFSService
   ) {
     super(options);
```

Comment:
Calling super(options) already sets this.options. It would be better to process the options before calling super instead of setting them twice.

Reply:
done

Reply:
done